src/ui/note/index.tsx

```diff
@@ -9,9 +12,9 @@ const Note = () => {
       </div>
 
       <div className="container-botoes">
-        <div>
-          <button>Editar</button>
-          <button>Excluir</button>
+        <div>s
+          <Link to='/edit'><button><img src={editarIcon} /></button></Link>
```

Comment:
A rota em `<Link to='/edit'>` já havia sido alterada. Isso significa que você deveria ter feito `pull` antes do `push`